pallets/corporate-actions/src/benchmarking.rs

```diff
@@ -0,0 +1,29 @@
+// This file is part of the Polymesh distribution (https://github.com/PolymathNetwork/Polymesh).
+// Copyright (c) 2020 Polymath
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use crate::*;
+use frame_benchmarking::benchmarks;
+use frame_system::RawOrigin;
+
+benchmarks! {
+    _ {}
+
+    set_max_details_length {}: _(RawOrigin::Root, 100)
+    verify {
+        assert_eq!(MaxDetailsLength::get(), 100);
```

Comment:
Could be replaced with `ensure!` to emit an error instead of an uninformative WASM panic backtrace.

Reply:
@vkomenda FWIW, for development purposes, you can run the benchmarks with `--execution Native` and that should give you more helpful rust backtraces.